src/datadoc/backend/core.py

```diff
@@ -45,7 +45,13 @@
 
 
 class Datadoc:
-    """Handle reading, updating and writing of metadata."""
+    """Handle reading, updating and writing of metadata.
+
+    Attributes:
```

Comment:
We should add a description if we include these here

Reply:
I actually work on improving docstrings in another branch, so I can remove the addition from this branch and the docstrings can be evaluated in another PR